src/api/main.ts

```diff
@@ -11,6 +11,7 @@ app.create(
   argv.uiBaseHref || '/',
   argv.inCluster === 'true',
   argv.namespace || 'default',
+  argv.forceNamespaceIsolation || 'false',
```

Comment:
Please change it to `argv.forceNamespaceIsolation === 'true'` . Otherwise UI always run in force namespace isolation mode.